Fix ISO name in release target

Found that ISO name hasn't been updated since we change its name to `minishift-centos7.iso`.

```
$make release
.... # iso building logs ....
mkdir -p /home/budhram/redhat/minishift-centos-iso/build
curl -sL https://github.com/progrium/gh-release/releases/download/v2.2.1/gh-release_2.2.1_linux_x86_64.tgz > /home/budhram/redhat/minishift-centos-iso/build/gh-release_2.2.1_linux_x86_64.tgz
tar -xvf /home/budhram/redhat/minishift-centos-iso/build/gh-release_2.2.1_linux_x86_64.tgz -C /home/budhram/redhat/minishift-centos-iso/build
gh-release
rm -fr /home/budhram/redhat/minishift-centos-iso/build/gh-release_2.2.1_linux_x86_64.tgz
rm -rf release && mkdir -p release
cp /home/budhram/redhat/minishift-centos-iso/build/minishift-centos.iso release/
cp: cannot stat '/home/budhram/redhat/minishift-centos-iso/build/minishift-centos.iso': No such file or directory
Makefile:73: recipe for target 'release' failed
make: *** [release] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -71,7 +71,7 @@ get_gh-release: init
 .PHONY: release
 release: centos_iso get_gh-release
 	rm -rf release && mkdir -p release
-	cp $(BUILD_DIR)/minishift-centos.iso release/
+	cp $(BUILD_DIR)/minishift-centos7.iso release/
 	$(BUILD_DIR)/gh-release checksums sha256
 	$(BUILD_DIR)/gh-release create minishift/minishift-centos-iso $(VERSION) master v$(VERSION)
 

